app-springboot/pom.xml

```diff
@@ -14,6 +14,7 @@
     <properties>
```

Comment:
Why are all these dependency changes needed? Does this indicate some breaking changes in our POMs?